client/assets/stylesheets/gutenboarding.scss

```diff
@@ -0,0 +1,23 @@
+// External Dependencies
+//@import 'vendor';
+
+// Color Schemes
+@import '~@automattic/calypso-color-schemes/src/calypso-color-schemes';
+
+// Shared
+@import 'shared/reset'; // css reset before the rest of the styles are defined
+
+// @TODO: import from `@wordpress/base-styles` package once it's available November 12 2019
+@import './gutenberg-base-styles/style';
+
+// Gutenberg resets
+// Not an actual "CSS reset", but an opiniated base style for HTML elements in Gutenberg views.
+:root {
+	@include reset;
+}
```

Comment:
Do we need this reset? How does it compare/relate to [the one in `client/landing/gutenboarding/style.scss`](https://github.com/Automattic/wp-calypso/pull/37141/files#diff-1762b786f4f0e0659667b586a2158493R10)?

Reply:
Oh yeah, we need to remove the one from Gutenboarding or move both resets there. Looks like I forgot to do that.

The thing is that Calypso reset (line 8 here) and Calypso reset need to be in the same file to ensure they load in that order because they overwrite each other.

Reply:
Removed duplicate in https://github.com/Automattic/wp-calypso/pull/37141/commits/908fe57685a38efc67d1c4738f31e5fabb171b2d